docs/usermanuals/tr/data/pii-ml.md

```
@@ -0,0 +1,140 @@
+| `person` | medium | Diğer bağlamla birlikte bir isim re-identify edebilir; tek başına ulusal kimlikten daha zayıf. |
+| `organization` | low | Kamu-kayıtlı varlıklar; sızıntı "bu kişi X'te çalışıyor" düzeyinde, "X'in ev adresi" değil. |
+| `location` | low | Aynı mantık — coğrafi string'ler tek başına genellikle de-identification-resistant'tır. |
+
+Bu katmanlar **bilinçli olarak** regex'in `critical`/`high` zeminlerinin (kredi kartları, ulusal kimlikler) altındadır. NER false-positive oranları regex-anchored detection'dan materyal olarak yüksektir; bu yüzden bir "person" bulgusu bir "credit_card" bulgusunun yaptığı gibi bir deployment'ı gate'lememeli.
+
+## Dil Seçimi {#dil-secimi}
+
+Non-English NLP engine'e işaret etmek için `--pii-ml-language` geçin. Talep edilen dil için bir engine kayıtlı değilse Presidio typed bir exception raise eder — eşleşen spaCy modelini kurabilmeniz için yüksek sesle yüzeye çıkar:
+
+```shell
+$ python -m spacy download xx_ent_wiki_sm     # çok dilli, daha küçük
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)